website/src/routes/apiv2.js

```diff
@@ -0,0 +1,776 @@
+/*
```

Comment:
Correction, it has been merged in, but it's not being used. The Content.coffee stuff is still being used.